core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/quiverPlot/QuiverPlotOpDesc.scala

```diff
@@ -0,0 +1,132 @@
+package edu.uci.ics.texera.workflow.operators.visualization.quiverPlot
+
+import com.fasterxml.jackson.annotation.{JsonProperty, JsonPropertyDescription}
+import com.kjetland.jackson.jsonSchema.annotations.{JsonSchemaInject, JsonSchemaTitle}
+import edu.uci.ics.texera.workflow.common.metadata.annotations.AutofillAttributeName
+import edu.uci.ics.texera.workflow.common.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.texera.workflow.common.operators.PythonOperatorDescriptor
+import edu.uci.ics.texera.workflow.common.tuple.schema.{Attribute, AttributeType, Schema}
+import edu.uci.ics.amber.engine.common.workflow.{InputPort, OutputPort}
+import edu.uci.ics.texera.workflow.operators.visualization.{
+  VisualizationConstants,
+  VisualizationOperator
+}
+
+@JsonSchemaInject(json = """
+{
+  "attributeTypeRules": {
+    "value": {
+      "enum": ["integer", "long", "double"]
+    }
+  }
+}
+""")
+class QuiverPlotOpDesc extends VisualizationOperator with PythonOperatorDescriptor {
+
+  @JsonProperty(value = "x", required = true)
+  @JsonSchemaTitle("x")
+  @JsonPropertyDescription("column for the x coordinate")
+  @AutofillAttributeName var x: String = ""
+
+  @JsonProperty(value = "y", required = true)
+  @JsonSchemaTitle("y")
+  @JsonPropertyDescription("column for the y coordinate")
+  @AutofillAttributeName var y: String = ""
+
```

Comment:
please add more explanation on what does `x` and `y` mean. Seems that they mean the starting point's coordinate

Reply:
Resolved